packages/console/src/cloud/pages/Main/Redirect.tsx

```diff
@@ -33,7 +34,7 @@ function Redirect({ tenants, toTenantId }: Props) {
     if (tenant) {
       void validate(tenant.indicator);
     }
-  }, [getAccessToken, href, setIsSettle, signIn, tenant]);
+  }, [getAccessToken, href, navigate, setIsSettle, signIn, tenant, toTenantId]);
```

Comment:
nit: The dependency `tenant` can be optimized to `tenant.indicator` only

Reply:
In line 34, need to check `tenant` existence.